clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_4_0.xsd

```diff
@@ -354,7 +354,7 @@
                         <xs:documentation>In ASYNC mode, this attribute controls how often the asynchronous thread used to flush the replication queue runs. This should be a positive integer which represents thread wakeup time in milliseconds.</xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
-                <xs:attribute name="remote-timeout" type="xs:long" default="17500">
+                <xs:attribute name="remote-timeout" type="xs:long" default="10000">
```

Comment:
We should actually change this in all the XSDs since whatever the XML schema version, the parsed default will be the new value.